src/main/java/org/squiddev/plethora/integration/rftools/IntegrationRFTools.java

```diff
@@ -0,0 +1,29 @@
+package org.squiddev.plethora.integration.rftools;
+
+import mcjty.rftools.RFTools;
+import org.squiddev.plethora.api.Injects;
+
+@Injects(RFTools.MODID)
+public final class IntegrationRFTools {
+	private IntegrationRFTools() {
+	}
+
+
+	/*
+	 *RFTools
+	 *
+	 *
+	 *
+	 *Control
+	 *
+	 *
+	 *
+	 *Dimensions
+	 * We could use IMC to get the Dimension Manager, a la `rftools.setup.ModSetup`,
```

Comment:
If we can pull it from IMC, that's probably the best route. The less we're relying on internal behaviour the better - McJTY does tend to refactor things a lot :).